docs/topics/modules/next_steps.adoc

```diff
@@ -3,15 +3,15 @@
 
 If you followed the instructions in this document, you have now learned about {osio} and its features. You are now able to:
 
-* Create Spaces for your projects
-* Add code from an existing project to your Space
-* Create new Vert.X and Spring Boot projects using the quickstart wizards and test them
-* Create Work Items to track your tasks and manage them
-* Use the build pipelines to stage reviews in the `Stage` and `Run` environments
-* Use Che workspaces to edit your project code and commit the changes to GitHub
-* Optimize your project memory usage to better use your OpenShift Online resources
-* Use Stack reports to make improvements to your code
-* Debug using the Che workspace
-* Clean up your {osio} account so you can start afresh
+* Create Spaces for your projects. See <<creating_new_space-hello-world>> for instructions.
```

Comment:
@mishaone, it all LGTM, but one nitpick: do we need to say "for instructions" at the end of each sentence? Well, it's not included everywhere, and I think it's sort of obvious, i.e. it can be omitted, which would make it look a bit leaner.